### PR TITLE
Patch for multiport stress test to handle iputils package issue

### PR DESCRIPTION
**Pull request by hathyaga**
This patch takes care of iputils package for distros

Signed-off-by: Harsha Thyagaraja <harshkid@linux.vnet.ibm.com>

### CODE REVIEW

**Comment by hathyaga** [comment]
Job log for Ubuntu distro
[joblog_multiport_run_Ubuntu1704.txt](https://github.com/avocado-framework-tests/avocado-misc-tests/files/977111/joblog_multiport_run_Ubuntu1704.txt)

**Comment by hathyaga** [comment]
Job log for Sles12 SP3  distro
[joblog_multiport_run_Sles12SP3.txt](https://github.com/avocado-framework-tests/avocado-misc-tests/files/977115/joblog_multiport_run_Sles12SP3.txt)

File: io/net/multiport_stress.py [hunk 1]
```diff
@@ -35,9 +36,13 @@ def setUp(self):
...
             self.skip("user should specify host interfaces")
         smm = SoftwareManager()
-        if not smm.check_installed("iputils-ping") and not\
-                smm.install("iputils-ping"):
-            self.skip("iputils-ping package is need to test")
+        if distro.detect().name == 'Ubuntu':
+            pkg = 'iputils-ping'
+        else:
+            pkg = 'iputils'
+        if not smm.check_installed(pkg) and not\
```
Comment (narasimhan-v) [inline io/net/multiport_stress.py]: why not `       if not smm.check_installed(pkg) and not smm.install(pkg):` ?

Comment (hathyaga) [inline io/net/multiport_stress.py]: I've used the same command as the comment.

Comment (narasimhan-v) [inline io/net/multiport_stress.py]: no need to wrap it to next line is what I meant

**Comment by hathyaga** [comment]
Change is made

**Comment by PraveenPenguin** [comment]
@narasimhan-v thanks

**Pull request merged**